src/AppBundle/Controller/ArticleController.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use AppBundle\Entity\Article;
+use AppBundle\Form\ArticleForm;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\BrowserKit\Response;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Class ArticleController.
+ */
+class ArticleController extends Controller
+{
+    /**
+     * @param $request
+     * @Route("/article/new", name="new_article")
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function newAction(Request $request)
+    {
+        $form = $this->createForm(ArticleForm::class);
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $em = $this->getDoctrine()->getManager();
+
+            $em->persist($form->getData());
```

Comment:
$form->getData can return not entity. it's bad for persist, you can persist only entity